### DISCUSSION TITLE
[resolve] Chinese characters in my calendar

### DISCUSSION THREAD

**Discussion by JC-LGMS** [valinet/ExplorerPatcher#2406] [Show and tell]
Just to share a problem and its solution.
I start to use ExplorerPatcher last week and I love it. Great job guys, thank you so much.
But I had one strange bug : I had Chinese characters appeared in my calendar (I setup my Windows only in French)
![image](https://github.com/valinet/ExplorerPatcher/assets/820334/242c392f-00bb-4d3a-8119-ca3e88d36ef8)

I search every where in this project but I didn't found the problem (nor the solution).
I search on Google and : https://techcommunity.microsoft.com/t5/windows-management/chinese-calendar-in-taskbar/m-p/1504248/thread-id/450

The simple answer : 
Go to Start > Settings > Time & Language > Date & Time. On the right side, scroll down to "Show additional calendar in taskbar". Make sure it is set to "Don't show additional calendar."
![image](https://github.com/valinet/ExplorerPatcher/assets/820334/03f3aaea-5a01-4101-a60a-76d08d62ade0)

**Comment by zems** [comment]
My friend, thank you!!! I was also looking for a solution to this problem, but I did not find it. Your advice worked 100%.

**Comment by Araichu** [comment]
this must be some sort of bug that still hasnt been fixed.. just got it. i searched it up, and all the other articles were absolutely useless, and then i found this :))
